vcGL/src/vcTexture.h

```diff
@@ -71,7 +71,7 @@ udResult vcTexture_AsyncCreateFromMemory(vcTexture **ppTexture, udWorkerPool *pP
 void vcTexture_Destroy(vcTexture **ppTexture);
 
 udResult vcTexture_UploadPixels(vcTexture *pTexture, const void *pPixels, int width, int height, int depth = 1);
-udResult vcTexture_SaveImage(vcTexture *pTexture, vcFramebuffer *pFramebuffer, const char *pFilename);
+udResult vcTexture_SaveImage(vcTexture *pTexture, vcFramebuffer *pFramebuffer, const char *pFilename, const char *pExtension);
```

Comment:
Use `udStrEndsWithi`, or use an enum to dictate the format regardless of the extension.